src/App.js

```diff
@@ -31,12 +36,16 @@ const appRouter = createBrowserRouter([
             },
             {
                 path: '/about',
-                element: <About />
+                element: <Suspense fallback={<h1>Loading...</h1>}><About /></Suspense>
             },
             {
                 path: '/contact',
                 element: <Contact />
             },
+            {
+                path: '/grocery',
+                element: <Suspense fallback={<h1>Loading...</h1>}><Grocery /></Suspense>
+            },
```

Comment:
using `Suspense` with fallback as we are using lazy,
the desired file has to be fetched from server through network call, which takes few millisec
but react is not bothered about it, In order to compensate it, and to avoid encountering error, in the mean time, react uses fallback component to be displayed in UI